src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/TwitterTests.java

```diff
@@ -31,45 +31,60 @@ public void prepareTest() {
   @Test(groups = "MercuryTwitterWidgetTest_001")
   @Execute(onWikia = "mercuryautomationtesting")
   public void MercuryTwitterWidgetTest_001_isLoadedOnFirstVisitDirectlyFromUrl() {
-    TwitterWidgetPageObject twitterWidget = new TwitterWidgetPageObject(driver);
+    TwitterWidgetPageObject widget = new TwitterWidgetPageObject(driver);
 
-    twitterWidget.createAndNavigate(wikiURL);
-    Assertion.assertTrue(twitterWidget.isLoadedOnMercury(), MercuryMessages.INVISIBLE_MSG);
+    widget.create().navigate(wikiURL);
+    Assertion.assertTrue(widget.isLoaded(), MercuryMessages.INVISIBLE_MSG);
   }
 
   @Test(groups = "MercuryTwitterWidgetTest_002")
   @Execute(onWikia = "mercuryautomationtesting")
   public void MercuryTwitterWidgetTest_002_isLoadedOnFirstVisitFromDifferentArticle() {
-    TwitterWidgetPageObject twitterWidget = new TwitterWidgetPageObject(driver);
+    TwitterWidgetPageObject widget = new TwitterWidgetPageObject(driver);
 
-    twitterWidget
+    widget
         .create()
         .openMercuryArticleByNameWithCbAndNoAds(wikiURL, MercurySubpages.MAIN_PAGE);
     new NavigationSideComponentObject(driver).navigateToArticle(TWITTER_ARTICLE_NAME);
 
-    Assertion.assertTrue(twitterWidget.isLoadedOnMercury(), MercuryMessages.INVISIBLE_MSG);
+    Assertion.assertTrue(widget.isLoaded(), MercuryMessages.INVISIBLE_MSG);
   }
 
   @Test(groups = "MercuryTwitterWidgetTest_003")
   @Execute(onWikia = "mercuryautomationtesting")
   public void MercuryTwitterWidgetTest_003_isLoadedOnSecondVisitFromDifferentArticle() {
-    TwitterWidgetPageObject twitterWidget = new TwitterWidgetPageObject(driver);
+    TwitterWidgetPageObject widget = new TwitterWidgetPageObject(driver);
 
-    twitterWidget.createAndNavigate(wikiURL);
+    widget.create().navigate(wikiURL);
 
     new NavigationSideComponentObject(driver)
         .navigateToArticle(MAPS_ARTICLE_NAME)
         .navigateToArticle(TWITTER_ARTICLE_NAME);
 
-    Assertion.assertTrue(twitterWidget.isLoadedOnMercury(), MercuryMessages.INVISIBLE_MSG);
+    Assertion.assertTrue(widget.isLoaded(), MercuryMessages.INVISIBLE_MSG);
+  }
+
+  @Test(groups = "MercuryTwitterWidgetTest_004")
+  @Execute(onWikia = "mercuryautomationtesting")
+  public void MercuryTwitterWidgetTest_004_areLoadedOnFirstVisitDirectlyFromUrl() {
+    TwitterWidgetPageObject widget = new TwitterWidgetPageObject(driver);
+
+    widget.createMultiple().navigate(wikiURL);
+
+    Assertion.assertTrue(
```

Comment:
Do you mean to write this in one line? It would not fit the max line chars boundaries. The autoformatter proposes this: 
```
    Assertion.assertTrue(twitterWidget.areAllValidSwappedForIFrames(),
                         MercuryMessages.SOME_VALID_WIDGETS_WERE_NOT_SWAPPED_MSG);
```